compiler/rustc_query_system/src/dep_graph/graph.rs

```diff
@@ -1417,15 +858,15 @@ struct LightGreenDepNodeData {
 /// `new_node_to_index` and `data`, or `prev_index_to_index` and `data`. When
 /// manipulating both, we acquire `new_node_to_index` or `prev_index_to_index`
 /// first, and `data` second.
-pub(super) struct CurrentDepGraph<K> {
-    data: Lock<DepNodeData<K>>,
+pub(super) struct CurrentDepGraph<K: DepKind> {
+    encoder: Steal<GraphEncoder<K>>,
     new_node_to_index: Sharded<FxHashMap<DepNode<K>, DepNodeIndex>>,
```

Comment:
Maybe rename this field to `node_to_current_index`.

Reply:
The name is still accurate: we only store nodes that did not exist in the previous session.

Reply:
:+1: